packages/paste-core/primitives/box/__tests__/StyleFunctions.test.tsx

```diff
@@ -0,0 +1,174 @@
+import {getPseudoStyles, getCustomElementStyles} from '../src/StyleFunctions';
+
+// When rendered the theme object is injected onto Box props via emotions theming context
+// which is where the css and variant style functions look for their values
+const mockTheme = {
+  theme: {
+    space: {
+      space10: '10px',
+    },
+    backgroundColors: {
+      colorBackgroundPrimary: 'blue',
+    },
+    borderColors: {
+      colorBorderError: 'red',
+    },
+    textColors: {
+      colorTextSuccess: 'green',
+    },
+    elements: {
+      CARD: {
+        color: 'colorTextSuccess',
+        borderColor: 'purple',
+        ':hover': {
+          cursor: 'pointer',
+        },
+        variants: {
+          primary: {
+            backgroundColor: 'colorBackgroundPrimary',
+            ':hover': {
+              margin: 'space10',
+              textDecoration: 'underline',
+            },
+          },
+        },
+      },
+      ALERT: {
+        padding: 'space10',
+        textDecoration: 'underline',
+        variants: {
+          foo: {
+            borderColor: 'colorBorderError',
+          },
+        },
+      },
+    },
+  },
+};
+
+describe('getPseudoStyles', () => {
+  it('should return resolved object styles when called with pseudo selector props set', () => {
+    const cssFunc = getPseudoStyles({_hover: {padding: 'space10', textDecoration: 'underline'}});
+    if (typeof cssFunc === 'function') {
```

Comment:
It's more to get around typing. `getPseudoStyles` returns either a css function or an empty object. I only want to test the css function here, but typescript is all "BUT IT MIGHT BE AN EMPTY OBJECT!!?!!"